library/ecdsa.c

```diff
@@ -37,11 +37,10 @@
 #include "mbedtls/asn1write.h"
 
 #include <string.h>
-
+#include "mbedtls/platform.h"
```

Comment:
Question: Why did you add this `#include` here?

Reply:
I think it's by mistake, when I added prints for debugginig. I'll remove